src/Slices/CreateEnvironment/UI/CreateEnvironmentForm.tsx

```diff
@@ -99,7 +112,7 @@ export const CreateEnvironmentForm: React.FC<Props> = ({ projects, ...props }) =
         label={words("createEnv.projectName")}
         value={projectName || ""}
         options={projects.map((project) => project.name)}
-        onCreate={createProject}
+        mutation={createProject}
```

Comment:
I wouldn't rename this onCreate event to something specific to React Query. The Select component should remain agnostic of the library we use to handle http and state.